### PR TITLE
Type room name to confirm close room

### PR DESCRIPTION
**Pull request by robinkwilson**
- Updated fsevents in package.json + package-lock.json because the old version is not hosted where npm expects -- likely deleted
- Textbox is difficult to see but matches the global styles
- If you type in a different input rather than the room name, a reddish border surrounds the textbox. 

closes https://github.com/mozilla/hubs/issues/2625

image

image

image

image

### CODE REVIEW

**Comment by robinkwilson** [comment]
The text box is really hard to see but matches the global styles.

File: src/assets/locales/en.json [hunk 1]
```diff
@@ -72,6 +72,7 @@
   "close-room.message": "Closing this room will remove yourself and others from the room, shutting it down permanently.\n\nAre you sure? This action cannot be undone.",
   "close-room.confirm": "Yes, Close Room",
   "close-room.cancel": "Cancel",
+  "close-room.type-to-confirm": "Type room name to confirm:",
```
Comment (brianpeiris) [inline src/assets/locales/en.json:75]: You'll need to run the `sync-locales.js` script to update the other locales with this new string.

File: src/react-components/close-room-dialog.js [hunk 2]
```diff
@@ -1,27 +1,54 @@
...
+        <input
+          className={classNames(
+            styles.formFieldText,
+            !isMatch && this.state.showIsNotMatchError ? styles.errorBorder : ""
+          )}
+          value={this.state.confirmText}
+          onChange={e => this.setState({ confirmText: e.target.value })}
+          required
+          spellCheck="false"
+          ref={inp => (this.nameInput = inp)}
```
Comment (brianpeiris) [inline src/react-components/close-room-dialog.js:41]: This `nameInput` ref is unused

**Review by brianpeiris — COMMENTED** [review]
Couple of comments. LGTM otherwise.

**Comment by brianpeiris** [comment]
Do we want to merge Elgin's change into this as well? #3450

**Pull request merged**